app/src/main/java/de/westnordost/streetcomplete/quests/cycleway/AddCycleway.kt

```diff
@@ -156,7 +157,7 @@ private val untaggedRoadsFilter by lazy { """
       and (
         !maxspeed
         or maxspeed > 20
-        or $notInZone30OrLess
+        or $onlyInANoneMaxSpeed30OrLessZone
```

Comment:
What about the `notInZone30OrLess` variable now?

Reply:
What do you mean? Is this about the name, too? Or about something else?